### PR TITLE
VTCCA-4713 remove empty cells

### PR DESCRIPTION
**Pull request by zahiriqbal**
https://jira.tools.tax.service.gov.uk/browse/VTCCA-4713

Minor screen change agreed between Karen and Zahir (add Action table header)

### CODE REVIEW

File: app/views/propertyrepresentation/manage/myAgents.scala.html [hunk 1]
```diff
@@ -56,6 +56,7 @@ <h1 class="govuk-heading-l">@messages("propertyRepresentation.showAgents.title")
             )
             ,
             HeadCell(
+                content = Text(messages("assessments.table.action"))
```
Comment (darrell-plant) [inline app/views/propertyrepresentation/manage/myAgents.scala.html:59]: You shouldn't reuse this message property. Create one for this page please.

Comment (darrell-plant) [reply app/views/propertyrepresentation/manage/myAgents.scala.html:59]: or one called 
```
label.action = Action
```

**Comment by darrell-plant** [comment]
LGTDz 👍

**Comment by DanSmith98** [comment]
LGTM :+1:

**Pull request merged**